File: hw/chicaiza/u3/hw26Guis/SchoolSystemPython/ec/edu/espe/schoolsystem/model/student_dao.py
from ec.edu.espe.schoolsystem.utils.mongo_connection import MongoConnection

class StudentDAO:

    def __init__(self):
        db = MongoConnection.get_database()
        self.collection = db["students"]

    def insert(self, student):
        document = {
            "id": student.id,
            "name": student.name,
            "pension": student.pension
        }
        self.collection.insert_one(document)

File: hw/chicaiza/u3/hw26Guis/SchoolSystem/src/main/java/ec/edu/espe/schoolsystem/model/StudentDAO.java

package ec.edu.espe.schoolsystem.model;

import com.mongodb.client.MongoCollection;
import com.mongodb.client.MongoDatabase;
import utils.MongoDBConnection;
import org.bson.Document;


/**
 *
 * @author Daniel
 */
public class StudentDAO {
    
    private MongoCollection<Document> collection;
      
    public   StudentDAO(){
        
        MongoDatabase database = MongoDBConnection.getDatabase();
        collection = database.getCollection("students");
        
    }
    
    public void insert (Student student){
        Document document = new Document("id",student.getId()).append("name", student.getName()).append("pension",student.getPension());
        collection.insertOne(document);
        
  
    }
    
}
